opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -707,6 +660,7 @@ where
 
 #[cfg(all(test, feature = "testing", feature = "trace"))]
 mod tests {
+    // cargo test trace::span_processor::tests:: --features=trace,testing
```

Comment:
nit: do we need this? the feature flag above seems to be clear to me what features needed to run this test

Reply:
I put it there just to make it easy for other devs to run these tests by copying and running the command, instead of having to type it manually every time! I use VSCode, and it does not offer any guided UI experience to run tests from a particular file.